src/main/java/uk/gov/hmcts/reform/sscs/service/SubmitAppealService.java

```diff
@@ -67,6 +64,16 @@ public Long submitAppeal(SyaCaseWrapper appeal) {
         return (caseDetails != null) ? caseDetails.getId() : null;
     }
 
+    public Long submitDraftAppeal(SyaCaseWrapper appeal) {
+        SscsCaseData caseData = convertSyaToCcdCaseData(appeal);
+
+        IdamTokens idamTokens = idamService.getIdamTokens();
+        SscsCaseDetails caseDetails = createCaseInCcd(caseData, EventType.DRAFT, idamTokens);
```

Comment:
let's duplicate this createCaseInCdd and rename something like createDraftInCcd so that you can have a specific implementation for the draft. the current createCaseInCcd takes care of duplication issues whereas the draft doesn't have to

Reply:
Duplicated.